Resources/config/token_providers.xml

```diff
@@ -8,6 +8,7 @@
         <parameter key="cmf_routing_auto.token_provider.content_method.class">Symfony\Cmf\Component\RoutingAuto\TokenProvider\ContentMethodProvider</parameter>
         <parameter key="cmf_routing_auto.token_provider.content_datetime.class">Symfony\Cmf\Component\RoutingAuto\TokenProvider\ContentDateTimeProvider</parameter>
         <parameter key="cmf_routing_auto.token_provider.content_locale.class">Symfony\Cmf\Component\RoutingAuto\TokenProvider\ContentLocaleProvider</parameter>
+        <parameter key="cmf_routing_auto.token_provider.container.class">Symfony\Cmf\Component\RoutingAuto\TokenProvider\SymfonyContainerParameterProvider</parameter>
```

Comment:
removing existing class parameters is a very nasty hidden BC, as things will just not behave as expected when people overwrote those parameters. we should do it eventually, but i guess as part of a 2.0 step - no urgency there.